src/services/graphql/mutation.ts

```diff
@@ -0,0 +1,10 @@
+import { gql } from "@apollo/client";
+
+export const createAuthor = gql`
```

Comment:
Nên viết UPPER_CASE cho const ở ngoài hàm, tương tự query
 
```suggestion
export const createAuthor = gql`
```